docs/user/ovirt/install_upi.md

```diff
@@ -43,16 +43,16 @@ mandatory ignition files and to monitor the installation process itself.
 * [Wait for Installation Complete](#wait-for-installation-complete)
 * [Destroy OpenShift cluster](#destroy-openshift-cluster)
 
-## Prerequisites
-The `inventory.yml` file all the variables used by this installation can be customized as per
+# Prerequisites
+The [inventory.yml](../../../upi/ovirt/inventory.yml) file all the variables used by this installation can be customized as per
 user needs.
 The requirements for the UPI in terms of minimum resources are broadly the same as 
 the [IPI](./install_ipi.md#minimum-resources).
 
-- oVirt/RHV account stored in the `ovirt-config.yaml`
+- oVirt/RHV account stored in the [ovirt-config.yaml](https://github.com/openshift/installer/blob/6abb3b5a8b687ee38b6c96368c77a305f6f0b563/docs/user/ovirt/install_ipi.md#ovirt-configyaml)
```

Comment:
The link points to a specific commit and will be outdated. IMHO the link should read https://github.com/openshift/installer/blob/master/docs/user/ovirt/install_ipi.md#ovirt-configyaml